src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagHandler.cs

```diff
@@ -30,9 +30,9 @@ public async Task<AddExtendedQueryTagResponse> Handle(AddExtendedQueryTagRequest
         {
             EnsureArg.IsNotNull(request, nameof(request));
 
-            if (await AuthorizationService.CheckAccess(DataActions.Write, cancellationToken) != DataActions.Write)
+            if (await AuthorizationService.CheckAccess(DataActions.Admin, cancellationToken) != DataActions.Admin)
```

Comment:
Should we name it to be more specific? "ManageExtendedQueryTags"?

Reply:
Not sure about this,  other actions in DataActions https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Core/Features/Security/DataActions.cs are very generic (read/write/delete)

Reply:
Admin feels overlapping to me. If you look at FHIR they had similar specific action for profiles I think